### DISCUSSION TITLE
Network Error

### DISCUSSION THREAD

**Discussion by nova98765** [requarks/wiki#6275] [Error / Bug Report]
When trying to save changes in pages that were created mid last year, I always get this error message:

`Network error: Unexpected token '<', "<!DOCTYPE "... is not valid JSON`

There is no `<` anywhere, we use Version 2.5.297 and we are using custom CSS

Does anyone know what could be causing such issue?

Edit: I noticed that it also appears with new pages when I try to edit cell properties.

**Comment by AdamB2023** [comment]
I'm getting the same error, running on IIS with Cloudflare, but already tried to disable all optimizations.
